docs/application/native/guides/device/system.md

```diff
@@ -26,15 +26,15 @@ To obtain the information, query a feature or system key.
 
```

Comment:
Line 1: System Information must be modified to sentence case - System information